proposals/0321-window-capabilities-template-name.md

```diff
@@ -31,11 +31,13 @@ The same change will be made to the HMI_API spec:
 <struct name="WindowCapability">
     <!-- Current params -->
     <param name="template" type="String" maxlength="500" mandatory="false">
-        <description>The name of the current template of this window. The other parameters describe the capabilities of this template layout.</description>
+        <description>Predefined or dynamically created window template. Currently only predefined window template layouts are defined.</description>
```

Comment:
```suggestion
        <description>Predefined or dynamically created window template. Currently, only predefined window template layouts are defined.</description>
```

Reply:
This would change to be different from the accepted revision and the other descriptions it's copying.